ouroboros-consensus-cardano-tools/src/Cardano/Tools/DBSynthesizer/Types.hs

```diff
@@ -0,0 +1,63 @@
+
+module Cardano.Tools.DBSynthesizer.Types (module Cardano.Tools.DBSynthesizer.Types) where
+
+import           Data.Aeson as Aeson (Value)
+import           Data.Word (Word64)
+
+import           Ouroboros.Consensus.Block.Abstract (SlotNo)
+import           Ouroboros.Consensus.Shelley.Eras (StandardShelley)
+import           Ouroboros.Consensus.Shelley.Node (ShelleyGenesis)
+
+import           Cardano.Node.Types (ProtocolFilepaths)
+
+
+data NodeConfigStub =
```

Comment:
We might want to make this alignment consistent with the style in the rest of the code base.